recipes/pykafka/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "pykafka" %}
+{% set version = "2.6.0" %}
+{% set sha256 = "19e4a802b00e28c3203fcfd80ca440b13faecb8e58171e9685a044cf87dae93e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - toolchain
```

Comment:
Please add `librdkafka` here too if it is needed for the build.

Reply:
Ah ah!  Of course.  Now I feel a bit silly.  Thanks @jakirkham !

Reply:
np